miso-web/src/main/webapp/scripts/list_kits.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+ * MISO project contacts: Robert Davey @ TGAC
+ * *********************************************************************
+ *
+ * This file is part of MISO.
+ *
+ * MISO is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MISO is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MISO.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * *********************************************************************
+ */
+
+ListTarget.kit = {
+  name : "Kits",
+  createUrl : function(config, projectId) {
+    return "/miso/rest/kitdescriptor/dt" + (config.kitType
+        ? "/type/" + config.kitType : "");
+  },
+  createBulkActions : function(config, projectId) {
+    
+    return [];
+  },
+  createStaticActions : function(config, projectId) {
+    return [ {
+      "name" : "Add",
```

Comment:
Oh ok. All the more reason to be explicit with the text on the Runs one then, since that does bring you to different pages.